### PR TITLE
Allow only one backend at a time

### PR DESCRIPTION
**Pull request by minottic**
relates to #107 and #115 

will have conflicts with #123

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,37 +26,44 @@ Note: older versions might not contain certain functionality (e.g. archival mock
...
 2. the SciCat [backend v3*](./services/backend/) container is created and connected to (1).
-3. the SciCat [backend v4*](./services/backendnext/) container is created and connected to (1).
-4. the SciCat [frontend](./services/frontend/) container is created and connected to (3).
-5. the SciCat [PaN searchapi](./services/searchapi/) container is created and connected to (3).
-6. a reverse [proxy](./services/proxy) container is created and routes trafic to (2), (3), (4) and (5) through localhost subdomains, in the form: `http://${service}.localhost` (for the ones of need). The frontend is available at simply `http://localhost`.
-
-We flag with `*` the services which have extra internal dependencies, which are not shared across the two backend versions, and with `**` the ones which have a dependency on the `BE_VERSION` value. To view them, refer to the service README.
-
+3. the SciCat [frontend**](./services/frontend/) container is created and connected to (2).
+4. the SciCat [PaN searchapi](./services/searchapi/) container is created and connected to (2).
```
Comment (consolethinks) [inline README.md:30]: I think, if we're running the backend next, then pan searchapi isn't used.

Comment (consolethinks) [inline README.md:30]: I think, if we're running the backend next, then pan searchapi isn't used.

File: README.md [hunk 2]
```diff
@@ -77,7 +84,7 @@ Accordingly,
 docker-compose up -d frontend(/searchapi)
 ```
 
-Will run, from the [previous section](#default-setup), (1), (2) and (4/(5)) but skip (3), (5/(4)) and 6.
+Will run, from the [previous section](#default-setup), (1), (2) and (3/(4)) but skip (4/(3)) and (5).
```
Comment (consolethinks) [inline README.md:87]: even if a little more verbose, but I think separating the two cases (`docker compose` up with `frontend` and `searchapi`) would make this more readable.

File: README.md [hunk 3]
```diff
@@ -94,11 +101,12 @@ To add a new service (see the [backend](./services/backend/) for an extensive ex
 2. call it as the service should be named
```
Comment (consolethinks) [inline README.md:101]: this part wasn't changed by this commit but I think "name the service" would be easier to understand

File: README.md (hunk 1 above)
Comment (consolethinks) [reply README.md:30]: Also, I don't see elastic search but maybe it could be "genericized" with pan search.

**Review by consolethinks — CHANGES_REQUESTED** [review]
Just a few corrections in the readme and it should be good afterwards.

File: README.md (hunk 2 above)
Comment (minottic) [reply README.md:87]: thanks, I will do the change!

**Review by consolethinks — APPROVED** [review]
seems all good

**Pull request merged**